### PR TITLE
feat: added home page

### PR DESCRIPTION
**Pull request by raman1999**
image
image

### CODE REVIEW

**Review by kalyandindu — APPROVED** [review]
Looks good to me.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            <p class="heading-4 l-sp-1 theme-shade-2">Whey Protein</p>
+        </div>
+        <div class="card">
+            <img class="card-img" alt="card-image" src="images/preworkout.jpg" />
+            <p class="heading-4 l-sp-1 theme-shade-2">Pre Workout</p>
+        </div>
+        <div class="card">
+            <img class="card-img" alt="card-image" src="images/dumbell.jpg" />
+            <p class="heading-4 l-sp-1 theme-shade-2">Gym accesories</p>
+
```
Comment (nishasen) [inline index.html]: remove the extra space

File: stylesheets/utilities.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    background-color: var(--theme-color);
+}
+.txt-theme{
+    color:var(--theme-color);
+}
+.txt-gray{
+    color:var(--txt-gray);
+}
+.txt-white{
+    color:#fff;
```
Comment (nishasen) [inline stylesheets/utilities.css]: use a variable here

**Review by nishasen — APPROVED** [review]
Your code is clean and good. Two small suggestions given

File: stylesheets/utilities.css (hunk 2 above)
Comment (raman1999) [reply stylesheets/utilities.css]: thanks, I will do it

**Pull request merged**